src/shared/infrastructure/typeorm/migrations/1682475482475-CreateUserTable.ts

```diff
@@ -0,0 +1,55 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class CreateUserTable1682475482475 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'users',
+        columns: [
+          {
+            name: 'id',
+            type: 'uuid',
+            isPrimary: true,
+            generationStrategy: 'uuid',
+            default: 'uuid_generate_v4()',
+          },
+          {
+            name: 'name',
+            type: 'varchar',
+          },
+          {
+            name: 'last_name',
```

Comment:
name and last name are necesssary in this case?

Reply:
yes, this information will be provided to the front end later on.